GMGoogleDrive/Public/New-GDriveFolder.ps1

```diff
@@ -30,6 +30,8 @@ param(
     [Parameter(Position=1)]
     [string[]]$ParentID = @('root'),
 
+    [DateTime]$CreationDate,

```

Comment:
maybe something like -DateTimeMeta since there is not only the creation date?

Reply:
Yes, we can use DateTimeMeta

Reply:
CreationDate changed to DateTimeMeta in 86ebf7a